Nighttime-Operations/Observatory/DIMM/Start-up.rst

```diff
@@ -0,0 +1,151 @@
+.. |author| replace:: *TDB*
+.. If there are no contributors, write "none" between the asterisks. Do not remove the substitution.
+.. |contributors| replace:: *none*
+
+.. _DIMM-StartUp:
+
+###############################
+DIMM Startup
+###############################
+
+The DIMM software automatically performs seeing measurements from dusk until dawn. Limited attendance from user is required. 
+
+The DIMM program will open the dome at dusk, start the system, slew the telescope, focus and offset the mount when needed, run the measurements, produce seeing values.  
+At dawn, the telescope will park itself, and the dome will close. This cycle repeats the following evening, if left unattended, when the sun sets.
+
+The `ameba` program is continously checking that the environment conditions are stable and clear, based on the data delivered by `tt-meteo`.
+In case of adverse weather, `ameba` will shut down the dome until conditions clear. 
+
+Currently, there is no incoming Cerro Pachón weather information feeding the `tt-meteo` program: Thus prior to starting up the DIMM, there are additional steps outlined in the :ref:`prerequisites section <Dimm_StartUp-Prerequisites>`.
+
+.. _Dimm_StartUp-Prerequisites: 
+
+Prerequisites: Weather data
+===============================
+
+To manually enter mock weather data into the `tt-weather` service so the DIMM disregards the lack of incoming weather data and commences operations, run:
+
+.. important::
+    Ensure the weather conditions are stable and safe before attempting the DIMM startup. 
+
+
+From a terminal in the LSST WAN network, connect to the DIMM VM:
+
+.. code-block:: bash
+
+        (base) you@Ios ~ % ssh dimm@139.229.191.103
+
+The password is located on the observers 1Password Vault.
+ 
+
+Connect via telnet to the `tt-meteo`. 
+
+.. code-block:: bash
+   
+        dimm@dimm:~$  telnet 127.0.0.1 16301
+
+
+The prompt within OpenTPL, a plain-text client-server protocol, is ``AUTH OK 0 0`` Commands are prepended with a freely chosen command id (unsigned integer) Here increasing integers (1,2) were used. 
+More on this communication protocol can be found on the :ref:`DIMM general specs <DIMM-DIMM>`.
+
+The following two commands can be written once the prompt appears. 
+
+.. code-block:: bash
+    
+        1 set sky.temp=-30
+    
+        2 set sky.status=0
+
+Quit telnet, pressing simultaneously ``Ctrl`` and  ``]``, write ``quit`` and click Enter. 
+
+The image below shows how the interaction with the OpenTPL client looks like and the two commands required to manually configure `tt-meteo`.
+
+.. figure:: ./_static/tt-meteo.png
+    :name: tt-meteo prerequisite
+
+    Prerequisite: Weather data
+
+Now the DIMM is ready to start operations. Follow only one of the procedures below, either start the DIMM up in :ref:`automatic mode <Dimm_StartUp-Automatic>` or in 
+:ref:`manual mode <Dimm_StartUp-Manual>`
+
+.. _Dimm_StartUp-Automatic: 
+
+Starting up the DIMM in automatic mode
```

Comment:
Rightfully or wrongfully, we've been capitalizing the letters of all major words in titles (I think)

Reply:
💯